dataset/get-versions.go

```diff
@@ -39,15 +39,31 @@ func getVersions(w http.ResponseWriter, req *http.Request, dc DatasetClient, use
 
 	log.Event(ctx, "calling get versions", log.Data(logInfo))
 
-	versions, err := dc.GetVersionsInBatches(ctx, userAccessToken, "", "", collectionID, datasetID, edition, batchSize, maxWorkers)
+	dataset, err := dc.GetDatasetCurrentAndNext(ctx, userAccessToken, "", collectionID, datasetID)
+	if err != nil {
+		errMsg := fmt.Sprintf("error getting dataset from dataset API: %v", err.Error())
+		log.Event(ctx, "error getting dataset from dataset API", log.ERROR, log.Error(err), log.Data(logInfo))
+		http.Error(w, errMsg, http.StatusInternalServerError)
+		return
+	}
+
+	edition, err := dc.GetEdition(ctx, userAccessToken, "", collectionID, datasetID, editionID)
+	if err != nil {
+		errMsg := fmt.Sprintf("error getting dataset from dataset API: %v", err.Error())
+		log.Event(ctx, "error getting dataset from dataset API", log.ERROR, log.Error(err), log.Data(logInfo))
```

Comment:
Should these log messages be "error getting edition"?

Reply:
Yup, C&P error. Thanks for spotting